src/styles/github.css

```diff
@@ -1,99 +1,98 @@
-/*
-
-github.com style (c) Vasily Polovnyov <vast@whiteants.net>
-
-*/
+/*!
+ * GitHub.com style
+ *
+ * GitHub Light v0.5.0
+ * https://github.com/primer/github-syntax-light
+ */
 
 .hljs {
   display: block;
   overflow-x: auto;
   padding: 0.5em;
-  color: #333;
-  background: #f8f8f8;
+  color: #24292e;
+  background: #ffffff;
 }
 
-.hljs-comment,
-.hljs-quote {
-  color: #998;
-  font-style: italic;
+.hljs-keyword,
+.hljs-doctag,
+.hljs-meta-keyword,
+.hljs-builtin-name,
+.hljs-type,
+.hljs-template-tag,
+.hljs-template-variable {
+  color: #d73a49;
 }
 
-.hljs-keyword,
-.hljs-selector-tag,
-.hljs-subst {
-  color: #333;
-  font-weight: bold;
+.hljs-title {
+  color: #6f42c1;
 }
 
-.hljs-number,
 .hljs-literal,
-.hljs-variable,
-.hljs-template-variable,
-.hljs-tag .hljs-attr {
-  color: #008080;
+.hljs-number,
+.hljs-attr,
+.hljs-attrib,
+.hljs-variab,
+.hljs-selector-id,
+.hljs-selector-class,
+.hljs-selector-attr {
+  color: #005cc5;
 }
 
+.hljs-regexp,
 .hljs-string,
-.hljs-doctag {
-  color: #d14;
+.hljs-meta-string {
+  color: #032f62;
 }
 
-.hljs-title,
-.hljs-section,
-.hljs-selector-id {
-  color: #900;
-  font-weight: bold;
+.hljs-built_in,
+.hljs-symbol,
+.hljs-class .hljs-title {
+  color: #e36209;
 }
 
-.hljs-subst {
-  font-weight: normal;
+.hljs-comment,
+.hljs-meta,
+.hljs-code,
+.hljs-formula {
+  color: #6a737d;
 }
 
-.hljs-type,
-.hljs-class .hljs-title {
-  color: #458;
+.hljs-section {
   font-weight: bold;
+  color: #005cc5;
 }
 
-.hljs-tag,
 .hljs-name,
-.hljs-attribute {
-  color: #000080;
-  font-weight: normal;
-}
-
-.hljs-regexp,
-.hljs-link {
-  color: #009926;
+.hljs-quote,
+.hljs-selector-tag,
+.hljs-selector-pseudo {
+  color: #22863a;
 }
 
-.hljs-symbol,
 .hljs-bullet {
-  color: #990073;
+  color: #735c0f;
 }
 
-.hljs-built_in,
-.hljs-builtin-name {
-  color: #0086b3;
+.hljs-emphasis {
+  font-style: italic;
+  color: #24292e;
 }
 
-.hljs-meta {
-  color: #999;
+.hljs-strong {
   font-weight: bold;
+  color: #24292e;
 }
 
-.hljs-deletion {
-  background: #fdd;
+.hljs-link {
+  text-decoration: underline;
```

Comment:
Does GitHub do this?

```md
This is a [link to google](http://www.google.com)...
```

Reply:
Are you perhaps confusing GitHub's markdown rendering with syntax coloring?  Unless you're seeing this somewhere else...?

Reply:
I think I was seeing this:
https://github.com/primer/github-syntax-light/blob/828ccb30b5a3a09b9b27bf52acaa90ac653d0fad/lib/github-light.css#L126-L129

Reply:
Removed.